src/client/js/otp/modules/planner/PlannerModule.js

```diff
@@ -362,6 +362,10 @@ otp.modules.planner.PlannerModule =
             if(this.numItineraries) queryParams.numItineraries = this.numItineraries;
             if(this.minTransferTime) queryParams.minTransferTime = this.minTransferTime;
             if(this.showIntermediateStops) queryParams.showIntermediateStops = this.showIntermediateStops;
+            if(this.watts) queryParams.watts = this.watts;
+            if(this.weight) queryParams.weight = this.weight;
+            if(this.minimumMicromobilitySpeed) queryParams.minimumMicromobilitySpeed = this.minimumMicromobilitySpeed;
```

Comment:
The reason I put minSpeed in here is to cover the case for when it would be possible to walk with the vehicle on grades so steep that the motor couldn't propel the vehicle up the hill. Like imagine if you had a kid's hoverboard with a weak motor... the speed calculations would say it'd only be able to go like 0.something miles per hour, but you could walk with the thing and still cover some ground. On the flipside, maybe you have a really heavy vehicle that you can't even push up a hill. In that case you'd want to set 0 as the minimum speed.